src/tools/pkcs15-tool.c

```diff
@@ -1675,6 +1704,34 @@ static void list_info(void)
 			count++;
 		}
 	}
+	printf("\n");
+	for (i = 0; i < SC_MAX_SUPPORTED_ALGORITHMS; i++) {
+		struct sc_supported_algo_info * sa = &p15card->tokeninfo->supported_algos[i];
```

Comment:
clang-format suggested the following change:


```
-		struct sc_supported_algo_info * sa = &p15card->tokeninfo->supported_algos[i];
+		struct sc_supported_algo_info *sa = &p15card->tokeninfo->supported_algos[i];
```

<!-- identifier: clang-format -->
<!-- File: src/tools/pkcs15-tool.c -->